multiqc_ngi/templates/ngi/header.html

```diff
@@ -10,7 +10,8 @@
 
 <h1 id="page_title">
   <a class="float-end" href="http://www.scilifelab.se/platforms/ngi/" target="_blank">
-    <img src="data:image/png;base64,{{ include_file('assets/img/NGI-final-small.png', b64=True) }}" title="National Genomics Infrastructure" style="max-width:320px;">
+    <img src="data:image/svg;base64,{{ include_file('assets/img/NGI_logo_20251110.svg', b64=True) }}" title="National Genomics Infrastructure" style="max-width:320px;" media="(prefers-color-scheme: light)">
+    <img src="data:image/svg;base64,{{ include_file('assets/img/NGI_logo_dark_mode_20251110.svg', b64=True) }}" title="National Genomics Infrastructure" style="max-width:320px;" media="(prefers-color-scheme: dark)"> 
```

Comment:
```suggestion
    <picture>
    <source srcset="data:image/svg+xml;base64,{{ include_file('assets/img/NGI_logo_dark_mode_20251110.svg', b64=True) }}" media="(prefers-color-scheme: dark)">
    <img src="data:image/svg+xml;base64,{{ include_file('assets/img/NGI_logo_20251110.svg', b64=True) }}" title="National Genomics Infrastructure" style="max-width:320px;" alt="NGI Logo">
</picture>
```
With this it also works in cases `prefers-color-scheme` is not supported

Reply:
Thanks @mashehu !